index.html

```diff
@@ -0,0 +1,125 @@
+                <p class="propara">Keynote speeches from experts in the VR industry can provide attendees with valuable insights into the latest trends, innovations, and challenges in the field.</p>  
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/panel.svg">
+                <h3 class="proheading">Panel discussions</h3>
+                <p class="propara">Panel discussions featuring experts in various aspects of VR can provide attendees with a deeper understanding of specific topics and allow them to hear different perspectives and insights.</p>               
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/training.jpg">
+                <h3 class="proheading">Workshops</h3>
+                <p class="propara">Workshops and training sessions can provide attendees with hands-on experience with VR technology and allow them to develop practical skills and knowledge.</p>                
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/vr.png">
+                <h3 class="proheading">VR experiences</h3>
+                <p class="propara">VR experiences and demonstrations can give attendees the opportunity to try out the latest VR hardware and software and experience immersive and interactive virtual environments.</p>               
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/networking.svg">
+                <h3 class="proheading">Networking events</h3>
+                <p class="propara">Networking events such as receptions, meet-and-greets, and informal gatherings can provide attendees with the opportunity to connect with other professionals in the VR industry, share ideas, and build relationships.</p>
+              </div>              
+            </div>
+            <div class="button-cintainer">
+            <button class="button">Join VR Odyssey 2023</button>
+           </div>
+           <div class="button-cintainer1">
+            <a class="button1" href="#">SEE THE WHOLE PROGRAM</a>
+           </div>
+        </section>
+        <section class="sponsers">
+          <h2 class="sponhead">Partners</h2>
+          <hr class="line4">
+          <ul class="sponicons">
+            <li class="sponicon">intel</li>
+            <li class="sponicon">ATHEER</li>
+            <li class="sponicon">Microsoft</li>
+            <li class="sponicon">Meta</li>
+            <li class="sponicon">Google</li>
+          </ul>
+        </section>
+        <footer class="footer">
+          <a class="footerlink1" href="#"><span class="footerlogo">VR Odyssey</span></a>
+          <div class="footerdiv"><i class="fa-solid fa-copyright footericon"></i>
+           <p class="footerpara">2023 vikas chib, All rights are Reserved<br>
+            Built with ❤️ by vikas</p>
+          </div>
```

Comment:
- [ ] You did well 😃 at adding the footer, but I see that the content of the footer in the Behance is on the left side little ,so please try to put the footer in the correct position like the Behance 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225889541-2964dda7-c5ac-4d49-b662-a4f6bcf50f1e.png)   |    ![image](https://user-images.githubusercontent.com/83360636/225889413-d364fceb-22a7-4788-9fb8-2819c422061f.png)  |